src/Glpi/Features/AssignableItemInterface.php

```diff
@@ -54,6 +54,8 @@ public static function getAssignableVisiblityCriteria(): array;
      */
     public function getRights($interface = 'central');
 
+    public function prepareGroupFields(array $input);
```

Comment:
Why not keeping the protected/private visibility ?

Reply:
Because the method need to be part of the interface, so it can be private.